test_runner/fixtures/zenith_fixtures.py

```diff
@@ -1170,6 +1170,34 @@ def test_output_dir(request: Any) -> str:
     return test_dir
 
 
+CONST_SKIP_FILES = frozenset(('pg_internal.init',
+                              'pg.log',
+                              'zenith.signal',
+                              'postgresql.conf',
+                              'postmaster.opts',
+                              'postmaster.pid',
+                              'pg_control'))
```

Comment:
Are we going to use this elsewhere? We could inline it if that's not the case.

Reply:
I would rather move directories to skip to top level also, IMO just cleaner this way even if not used anywhere else